mlos_bench/mlos_bench/config/schemas/services/remote/azure/azure-flex-service-subschema.json

```diff
@@ -39,12 +39,28 @@
                                 "Microsoft.DBforPostgreSQL",
                                 "Microsoft.DBforMariaDB"
                             ]
+                        },
+                        "supportsBatchUpdate": {
+                            "description": "Whether API supports updating multiple config parameters at once",
+                            "type": "boolean"
+                        },
+                        "isFlex": {
+                            "description": "Whether SaaS is an Azure Flexible Server",
```

Comment:
Also provide a documentation URL in the description for additional details?

Reply:
yep